bin/kano-dev

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+
+# kano-dev
+#
+# Copyright (C) 2016 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+# Run common dev commands. Expects to run as root
+
+"""
+kano-dev runs common dev commands on the pi
+
+Usage:
+  kano-dev rootssh
+  kano-dev production (all|api|apt)
+  kano-dev staging (all|api|apt)
+  kano-dev scratch
+
+ Options:
+"""
+
+import docopt
+import os
+
+KANO_CONTENT_STAGING = "api-url: http://kano-content-api-test.herokuapp.com\n"
+
+KANO_WORLD_STAGING = """api_url: http://kano-api-staging.herokuapp.com
```

Comment:
Well, we never really changed, the api-staging.kano.me redirects to the right instance, but given that there were some proxying issues with it (that are hopefully now resolved) we fell back to using the urls of the actual staging instances.

Just to be 100% sure that all the issues are resolved, I would greatly appreciate if you could test with the new url locally and let me know if you encountered any issues.

I don't think there have been any changes on the content api, maybe @pazdera can verify.